openfe/protocols/openmm_utils/multistate_analysis.py

```diff
@@ -0,0 +1,619 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+"""
+Reusable utility methods to analyze results from multistate calculations.
+"""
+from pathlib import Path
+from typing import Union, Optional
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.typing as npt
+from openmmtools import multistate
+from openff.units import unit, ensure_quantity
+
+
+class MultistateEquilFEAnalysis:
```

Comment:
I'm not really a fan of another class.  Anything class-like should probably get rolled into the existing `ProtocolUnitResult`, anything that is useful for working with .nc file's energies should just be a function.